src/components/executive-education-2u/ExecutiveEducation2UError.jsx

```diff
@@ -47,9 +76,19 @@ const ExecutiveEducation2UError = ({ failureReason, httpReferrer }) => {
                 global.location.href = httpReferrer;
               }}
             >
-              Return to your learning platform
+              Return to dashboard
             </Button>
           )}
+          {showHelpfulLink(failureReason) && (
+            <div>
+              <span className="executive-education-2u-error-link-description">
+                Helpful Link:
+              </span>
+              <Chip className="executive-education-2u-error-link-chip">
+                <ContactAdminMailto />
+              </Chip>
```

Comment:
Is there an alternative to using `Chip` here? It's not really being used in a way that `Chip` intends (i.e., using it to contain a hyperlink, which requires style overrides). Does the helpful link _need_ to rendered inside of a `Chip` and/or should it look more link-like according to our design system (e.g., the default styles for `MailtoLink` or `<Button variant="link" size="...">...</Button>`)?

https://paragon-openedx.netlify.app/components/chip/

Reply:
I removed the use of `Chip` component and used the `Button` component with link variant and added the box borders using CSS.